def convery_number(s):
    try:
        return int(s)
    except ValueError:
        return None
#you put the code you want to try inside the try block.
